ce/ce/artifacts/activation.ts

```diff
@@ -607,15 +536,13 @@ export class Activation {
     }
 
     // .defines get compiled into a single environment variable.
-    if (this.definesCount) {
-      let defines = '';
-      for await (const [name, value] of this.defines) {
-        defines += value ? `-D${name}=${value} ` : `-D${name} `;
-      }
-      if (defines) {
-        env['DEFINES'] = defines;
-        undo['DEFINES'] = originalEnvironment['DEFINES'] || '';
-      }
+    let defines = '';
+    for await (const [name, value] of this.defines) {
+      defines += value ? `-D${name}=${value} ` : `-D${name} `;
```

Comment:
If value is `0`, does this work correctly?

Reply:
I have no idea. This isn't actually edited though. See previous line 613.